scripts/gen_ed25519_key.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+"""Generate an ed25519 keypair, and store as base64-encoded text files.
+
+This script doesn't currently reuse the functions in `scriptworker.ed25519`, for
+easier standalone use. It could easily be a `console_script` though.
+
+"""
+from __future__ import print_function
+import base64
+from nacl.signing import SigningKey, VerifyKey
+import sys
+
+
+def to_string(key):
+    """Convert an ed25519 key to a base64-encoded string."""
+    return base64.encodestring(bytes(key)).strip().decode('utf-8')
```

Comment:
https://pynacl.readthedocs.io/en/stable/encoding/ suggests that we don't need to implement these ourself.

Reply:
Nice, thanks for the pointer!